src/tests/compositor/Compositor_Geometry_001.test.js

```diff
@@ -0,0 +1,55 @@
+import {
+  pluginDeactivate,
+  pluginActivate,
+  getClientGeometry,
+  setClientGeometry,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'Compositor Geometry - 001',
+  description: 'Sets and gets the Geometry of client',
+  steps: [
+    {
+      description: 'Deactivate Netflix Plugin and check deactivated or not',
+      test: pluginDeactivate,
+      params: constants.netFlixPlugin,
```

Comment:
same here, switch to WebKitBrowser or UX

Reply:
Updated code as per the review comment